deployment/CP01CNSM_Deploy.csv

```diff
@@ -1,409 +1,443 @@
+AR48,,,,CP01CNSM-MFD37-01-OPTAAD000,13,1,2020-10-28T00:00:00,,CGMCP-01CNSM-00013,,CGINS-OPTAAD-00151,40,-70,,,,
+AR48,,,,CP01CNSM-MFD37-03-CTDBPD000,13,1,2020-10-28T00:00:00,,CGMCP-01CNSM-00013,,CGINS-CTDBPD-07209,40,-70,,,,
+AR48,,,,CP01CNSM-MFD37-04-DOSTAD000,13,1,2020-10-28T00:00:00,,CGMCP-01CNSM-00013,,CGINS-DOSTAD-00390,40,-70,,,,
+AR48,,,,CP01CNSM-MFD37-07-ZPLSCC000,13,1,2020-10-28T00:00:00,,CGMCP-01CNSM-00013,,CGINS-ZPLSCC-55103,40,-70,,,,
+AR48,,,,CP01CNSM-RIC21-00-CPMENG000,13,1,2020-10-28T00:00:00,,CGMCP-01CNSM-00013,,CGCON-NCPM02-00011,40,-70,,7,,
+AR48,,,,CP01CNSM-RID26-00-DCLENG000,13,1,2020-10-28T00:00:00,,CGMCP-01CNSM-00013,,CGCON-NDCL26-00031,40,-70,,7,,
+AR48,,,,CP01CNSM-RID26-04-VELPTA000,13,1,2020-10-28T00:00:00,,CGMCP-01CNSM-00013,,CGINS-VELPTA-11402,40,-70,,7,,
+AR48,,,,CP01CNSM-RID26-06-PHSEND000,13,1,2020-10-28T00:00:00,,CGMCP-01CNSM-00013,,CGINS-PHSEND-P0155,40,-70,,7,,
+AR48,,,,CP01CNSM-RID26-07-NUTNRB000,13,1,2020-10-28T00:00:00,,CGMCP-01CNSM-00013,,CGINS-NUTNRB-01064,40,-70,,7,,
+AR48,,,,CP01CNSM-RID26-08-SPKIRB000,13,1,2020-10-28T00:00:00,,CGMCP-01CNSM-00013,,CGINS-SPKIRB-00230,40,-70,,7,,
+AR48,,,,CP01CNSM-RID27-00-DCLENG000,13,1,2020-10-28T00:00:00,,CGMCP-01CNSM-00013,,CGCON-NDCL27-00011,40,-70,,7,,
+AR48,,,,CP01CNSM-RID27-01-OPTAAD000,13,1,2020-10-28T00:00:00,,CGMCP-01CNSM-00013,,CGINS-OPTAAD-00123,40,-70,,7,,
+AR48,,,,CP01CNSM-RID27-02-FLORTD000,13,1,2020-10-28T00:00:00,,CGMCP-01CNSM-00013,,CGINS-FLORTD-01215,40,-70,,7,,
+AR48,,,,CP01CNSM-RID27-03-CTDBPC000,13,1,2020-10-28T00:00:00,,CGMCP-01CNSM-00013,,CGINS-CTDBPC-06841,40,-70,,7,,
+AR48,,,,CP01CNSM-RID27-04-DOSTAD000,13,1,2020-10-28T00:00:00,,CGMCP-01CNSM-00013,,CGINS-DOSTAD-00389,40,-70,,7,,
+AR48,,,,CP01CNSM-SBC11-00-CPMENG000,13,1,2020-10-28T00:00:00,,CGMCP-01CNSM-00013,,CGCON-BCPM01-00009,40,-70,,0,,
+AR48,,,,CP01CNSM-SBD11-00-DCLENG000,13,1,2020-10-28T00:00:00,,CGMCP-01CNSM-00013,,CGCON-BDCL11-00009,40,-70,,0,,
+AR48,,,,CP01CNSM-SBD11-01-MOPAK0000,13,1,2020-10-28T00:00:00,,CGMCP-01CNSM-00013,,CGINS-MOPAK0-43264,40,-70,,0,,
+AR48,,,,CP01CNSM-SBD11-02-HYDGN0000,13,1,2020-10-28T00:00:00,,CGMCP-01CNSM-00013,,CGINS-HYDGN0-11407,40,-70,,0,,
+AR48,,,,CP01CNSM-SBD11-06-METBKA000,13,1,2020-10-28T00:00:00,,CGMCP-01CNSM-00013,,CGINS-METLGR-00003,40,-70,,0,,
+AR48,,,,CP01CNSM-SBD12-00-DCLENG000,13,1,2020-10-28T00:00:00,,CGMCP-01CNSM-00013,,CGCON-BDCL12-00009,40,-70,,0,,
+AR48,,,,CP01CNSM-SBD12-03-HYDGN0000,13,1,2020-10-28T00:00:00,,CGMCP-01CNSM-00013,,CGINS-HYDGN0-49860,40,-70,,0,,
+AR48,,,,CP01CNSM-SBD12-04-PCO2AA000,13,1,2020-10-28T00:00:00,,CGMCP-01CNSM-00013,,CGINS-PCO2AA-33183,40,-70,,0,,
+AR48,,,,CP01CNSM-SBD12-05-WAVSSA000,13,1,2020-10-28T00:00:00,,CGMCP-01CNSM-00013,,CGINS-WAVSSA-06271,40,-70,,0,,
+AR48,,,,CP01CNSM-SBD12-06-METBKA000,13,1,2020-10-28T00:00:00,,CGMCP-01CNSM-00013,,CGINS-METLGR-00007,40,-70,,0,,
+AR48,,,,CP01CNSM-SBD12-08-FDCHPA000,13,1,2020-10-28T00:00:00,,CGMCP-01CNSM-00013,,CGINS-FDCHPA-75008,40,-70,,-5,,
```

Comment:
Looks good. Compared serial numbers to config sheet and confirmed CUID changes

Reply:
Looks good.